utils/clients/client.go

```diff
@@ -86,6 +86,8 @@ func NewWithProxy(name string, serverURL string, authToken string, proxyURL stri
 			Transport: middleware.InstrumentRoundTripper(
 				&http.Transport{
 					Proxy: proxy,
+					// can i just remove this?
+					// DialTLS: pindialer.MakeDialer(upholdCertFingerprint),
```

Comment:
Yes, this is only valid for the uphold client afaik, which is created here: https://github.com/brave-intl/bat-go/blob/master/utils/wallet/provider/uphold/uphold.go#L120

Reply:
super. removed

Reply:
fixed in 7f0ed18